docs/docs/tutorial/part-2/index.mdx

```diff
@@ -758,6 +758,12 @@ Luckily, when you use CSS Modules with Gatsby, you can have both! Your kebab-cas
 
 ![A screenshot of the Home page. The styles have been updated so now the page title is in purple, and the navigation links at the top of the page are in a single line instead of a bulleted list.](./index-page-styled.png)
 
+<Announcement style={{marginBottom: "1.5rem"}}>
+
+**Want to see how it all fits together?** Check out the commit history in the [GitHub repo for the finished example site](https://github.com/gatsbyjs/tutorial-example).
+
+</Announcement>
```

Comment:
Definitely not blocking, but you can put something like this into its own MDX file to turn it into a component, then import it places like this and in the other parts, e.g. `import CommitHistoryCallout from './commit-history-callout.mdx`

Another thought I had when looking at this: if you have 1-to-1 commits with each step, you could link directly to that commit in GitHub

Reply:
Oh nice, I'd never thought about importing from other MDX files.

Also I thought about adding links to the exact commits, but I've been rebasing and force-pushing the demo site when I make tweaks to the Tutorial content, which messes up all the commit hashes. I figured it would be less work to maintain to use a generic link to the repo. (Unless maybe there's some git trick for updating branches when you rebase?)